lib/redcord.rb

```diff
@@ -1,9 +1,36 @@
+# frozen_string_literal: true
+
 # typed: strict
-module Redcord
-end
 
 require 'sorbet-runtime'
```

Comment:
nit: can we keep these require statements together? Unless you separated them for a particular reason

Reply:
yes, we used sorbet first in this file. Then we'd need to define module Redcord prior to other files